docs/api/session.md

```diff
@@ -144,6 +144,70 @@ Emitted when a hunspell dictionary file download fails.  For details
 on the failure you should collect a netlog and inspect the download
 request.
 
+#### Event: 'select-serial-port' _Experimental_
+
+Returns:
+
+* `event` Event
+* `portList` [SerialPort[]](structures/serial-port.md)
+* `webContents` [WebContents](web-contents.md)
+* `callback` Function
+  * `portId` String
+
+Emitted when a serial port needs to be selected when a call to
+`navigator.serial.requestPort` is made. `callback` should be called with
+`portId` to be selected, passing an empty string to `callback` will
+cancel the request.  Additionally, permissioning on `navigator.serial` can
+be managed by using [ses.setPermissionCheckHandler(handler)](#sessetpermissioncheckhandlerhandler)
+with the `serial` permission.
+
+Because this is an experimental feature it is disabled by default.  To enable this feature, you
+will need to use the following command line switches: `--enable-experimental-web-platform-features` and
+`enable-features=ElectronSerialChooser`.
+
+```javascript
+const { app, BrowserWindow } = require('electron')
+
+let win = null
+app.commandLine.appendSwitch('enable-experimental-web-platform-features')
```

Comment:
we should probably recommend per-WebContents flagging of this feature via the `experimentalFeatures` WebPreference. Further, is there a way to enable _just_ the serial API rather than all experimental features?

Reply:
Ah yes there is `--enable-blink-features=Serial` or `enableBlinkFeatures: 'Serial'` via webpreference.